core/remote/cozy.js

```diff
@@ -219,7 +219,7 @@ class RemoteCozy {
   }
 
   // Retrieve the path of a remote file doc
-  async _setPath (doc /*: any */) /*: Promise<void> */ {
+  async _setPath (doc /*: * */) /*: Promise<*> */ {
```

Comment:
Can't we use `RemoteDoc` to type `doc`?

Reply:
Nope, it is called with things that looks like `RemoteDoc` but are not: they don't have a `path`, that is mandatory for `RemoteDoc`